pyopia/background.py

```diff
@@ -109,17 +111,19 @@ def correct_im_fast(imbg, imraw):
     Args:
       imbg (float64)  : background averaged image
       imraw (float64) : raw image
+      imbg (float64)  : background averaged image
+      imraw (float64) : raw image
 
     Returns:
-      imc (float64)   : corrected image
+      im_corrected (float64)   : corrected image
     '''
-    imc = imraw - imbg
+    im_corrected = imraw - imbg
 
-    imc += 215/255
-    imc[imc < 0] = 0
-    imc[imc > 1] = 1
+    im_corrected += 215/255
+    im_corrected[im_corrected < 0] = 0
+    im_corrected[im_corrected > 1] = 1
```

Comment:
123-124 could be a one-liner with numpy: im_corrected = np.clip(im_corrected, 0, 1)